.github/docker/testing/Dockerfile.testing-plugins-alma8

```diff
@@ -60,9 +60,14 @@ gpgcheck=1\n\
 gpgkey=https://yum-gpg.centreon.com/RPM-GPG-KEY-CES\n'\
 >> /etc/yum.repos.d/centreon-plugins.repo
 
+dnf config-manager --add-repo https://packages.centreon.com/rpm-standard/24.10/el8/centreon-24.10.repo
```

Comment:
does it have to be 24.10 or can it be any repository serving centreon-connector-perl and centreon-clib ?

Reply:
it can be any repository